### ISSUE TITLE
Date/Time Survey Answer Type

### ISSUE THREAD

**Issue by jamesmarshall24** [ansible/awx#4307]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY

Add a built-in answer type for date and time that translates to a usable timestamp string for job runs

**Comment by rebeccahhh** [comment]
@jamesmarshall24 are you hoping this built-in to still prompt the user the way a survey with other answer types would?

**Comment by eschek87** [comment]
This is a feature that we need, too!

**Comment by gordonAndersen** [comment]
Yes, it would be super to have a date/time field. Supporting the format Y-M-D H-M-SZ
